geth/api/backend_jail_test.go

```diff
@@ -279,6 +283,8 @@ func (s *BackendTestSuite) TestJailWhisper() {
 	jailInstance := s.backend.JailManager()
 	require.NotNil(jailInstance)
 
+	jailInstance.BaseJS(baseStatusJSCode)
```

Comment:
By the way, there wasn't such thing before. Why did we begin to need it?

Reply:
At the time, this was added due to complains of `ssh not defined`, I have removed it on my changes but still facing abit of a json deserialization error on the go side within ethereum. Need to investiguate further and fix then push up.

Reply:
This is actually needed to load up the `_status_catalog_` stand in for the test